### DISCUSSION TITLE
Редактирование конфига Amnezia docker-compose.override.yml

### DISCUSSION THREAD

**Discussion by tunivi** [xtrime-ru/antizapret-vpn-docker#76] [Q&A]
На МТС стали блокировать Amnezia, поэтому захотел исправить параметры JC, JMIN, JMAX в конфиге сервера. Внёс в файл `docker-compose.override.yml` следующие строки:

>       - UI_TRAFFIC_STATS=true
>       - UI_ENABLE_SORT_CLIENTS=true
>       - JC=4
>       - JMIN=40
>       - JMAX=70
>       - S1=3
>       - S2=7

Но в итоге после сборки контейнера сработали только переменные  `UI_TRAFFIC_STATS` и `UI_ENABLE_SORT_CLIENTS`, направленные на интерфейс веб-панели. `JC`, `JMIN`, `JMAX` в конфиге `wg0.conf` остались прежними.  Также не срабатывает переменная `WG_HOST`. Докер для меня пока тёмный лес, помогите пожалуйста.

**Comment by xtrime-ru** [comment]
Все выглядит корректно. Возможно надо пересоздать контейнер что бы применились новые параметры:
docker-compose up -d amnezia-wg-easy --force-recreate
